hack/test-unit.sh

```diff
@@ -5,7 +5,6 @@ os::log::info "Running unit tests"
 
 
 PACKAGES_TO_TEST=(
-    "github.com/openshift/console-operator/pkg/apis/console/v1"
```

Comment:
It looks like the tests below might be fixed. Is this change still needed?

Reply:
Yup, this package no longer exists.

Reply:
Right! I see now, thanks